src/sentry/templates/sentry/emails/reports/body.html

```diff
@@ -363,7 +363,7 @@ <h1>
 
     {% with series=report.projects.series height=110 %}
 
-    <h4>Errors by Project</h4>
+    <h4>Events by Project</h4>
```

Comment:
@Neo-Zhixing if we aren't including transactions in this yet, I think we should just leave the title as errors